Libraries/Experimental/Incremental.js

```diff
@@ -94,6 +94,11 @@ export type Props = {
 };
 type State = {doIncrementalRender: boolean, ...};
 
+const IncrementalContext = React.createContext<Context>({
```

Comment:
This component is not a context provider, but a consumer, so there's no need to create a new context here. 
@TheSavior I guess this component is used internally at Facebook or it's some dead code that haven't been removed yet. Can you shed some information on the topic? If it can be removed, that's great. If not, we need to get the context from somewhere.